### PR TITLE
KDE KF6 update

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

**Comment by travier** [comment]
Looks like it's working (did not render anything).

But I got:
```
Missing MLT module: kdenlivetitle
required to create titles
Missing MLT module: glaxnimate
required to load Lottie animations
```

**Comment by travier** [comment]
bot, build

**Comment by travier** [comment]
bot, build

**Comment by travier** [comment]
bot, build

**Comment by jakubklos77** [comment]
Unfortunately, this version has the "Add Title Clip" disabled (greyed out). If I use the official, it works and the menu item is enabled. So I cannot test the functionality in this build...

**Comment by jlskuz** [comment]
You need to update the cmake flags for MLT like:

```
        "-DMOD_OPENCV=ON",
        "-DMOD_MOVIT=ON",
        "-DMOD_GLAXNIMATE=OFF",
        "-DMOD_QT=OFF",
        "-DMOD_QT6=ON",
        "-DMOD_GLAXNIMATE_QT6=ON",
```

File: org.kde.kdenlive.json [hunk 1]
```diff
@@ -852,7 +853,7 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "http://git.sesse.net/movit",
+                    "url": "https://git.sesse.net/movit",
```
Comment (frdbr) [inline org.kde.kdenlive.json:856]: Movit support has been temporarily removed in Kdenlive for the time being.

Comment (jlskuz) [reply org.kde.kdenlive.json:856]: Yes, we could remove it from the dependencies with 24.02

**Comment by travier** [comment]
Thanks, updated. Let's see how this goes.

**Comment by jakubklos77** [comment]
Yes, that build works for me including the rendering. Thank you

**Comment by gtirloni** [comment]
Also confirming this works great here (Fedora 39 x86-64).

**Comment by travier** [comment]
Thanks a lot for testing!

**Comment by travier** [comment]
OK, this looks good for a quick test. @jlskuz Can I let you merge if you think this is good? Thanks

**Pull request merged**